provider/huggingchat.py

```diff
@@ -41,9 +42,19 @@ def instruct(self, prompt: str) -> str:
         )
         try:
             data = res.json()
+            data = data[0] if data else {}
         except ValueError:
             print("Invalid JSON response")
             data = {}
+        except:
+            if data.get("error_type", None) == "overloaded":
```

Comment:
Actually, now that I think of it. If data doesn't have error_type, this will throw an error